gabby/extracts/view/gsheets_comp_events.sql

```diff
@@ -44,6 +44,17 @@ WITH
         )
       ) AS p
   ),
+  campus_approval AS (
+    SELECT
+      p.campus_name,
+      p.region,
+      s.manager_df_employee_number AS mdso
```

Comment:
add `employee_number` to the name for clarity

Reply:
i need just the mdso associated with a site and not the DSO, because people campus sites don't have dsos. there's no way to link someone who works at a campus to an mdso besides looking for a dso at the same campus but at a school and then looking for their manager (the MDSO)

Reply:
I mean the alias

Reply:
ah ok, done.